test/e2e/specs/fakerp/keyrotation.go

```diff
@@ -2,28 +2,19 @@ package fakerp
 
 import (
 	"context"
-	"flag"
-	"io/ioutil"
 	"net/http"
 	"os"
 	"reflect"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
-	"github.com/ghodss/yaml"
-
-	"github.com/openshift/openshift-azure/pkg/api"
-	fakerp "github.com/openshift/openshift-azure/pkg/fakerp/client"
-	"github.com/openshift/openshift-azure/pkg/util/managedcluster"
 	"github.com/openshift/openshift-azure/test/clients/azure"
 )
 
 var _ = Describe("Key Rotation E2E tests [KeyRotation][Fake][LongRunning]", func() {
 	var (
-		cli        *azure.Client
-		manifest   = flag.String("manifest", "../../_data/manifest.yaml", "Path to the manifest to send to the RP")
```

Comment:
You need to update the test requirements in parallel to this change.

Reply:
ok